packages/amplication-client/src/Workspaces/ResourceList.tsx

```diff
@@ -62,6 +63,16 @@ function ResourceList() {
     },
   });
 
+  const handleSelect = useCallback(
```

Comment:
you can add the track event inside useResource hook and add setResource directly to resourceListItem instead of passing it two levels